<launch>
<node name="tf_listener" pkg="robot_setup_tf" type="tf_listener"  output="screen"/>
<node name="tf_broadcaster" pkg="robot_setup_tf" type="tf_broadcaster"  output="screen"/>
<param name="base_local_planner" value="local_planner/LocalPlanner" />
</launch>